misc/windows-deploy/amazon-ecs-agent.ps1

```diff
@@ -79,18 +91,18 @@ try {
             docker build -t amazon/amazon-ecs-credential-proxy --file .\credentialproxy.dockerfile . | out-null
             docker run --name ecs-cred-proxy -d -p 80:51679 amazon/amazon-ecs-credential-proxy | out-null
         } catch {
-            LogMsg -message "Running Credential Proxy container failed" -logLevel "INFO"
+            LogMsg -message "Running Credential Proxy container failed." -logLevel "ERROR"
```

Comment:
I don't think that's unreasonable, but I also don't think it's necessary for now.